README.md

```diff
@@ -1,2 +1,29 @@
-## DAT255
-Course-related documentation can be found in the `/documents`-branch
+## DAT255 - Group Smiley
+
+Course-related documentation can be found in the `/documents`-branch (e.g. Reflection Report)
+
+`/doc` folder contains code-specific documentation (e.g. TCP-protocol)
```

Comment:
It contains documentation that is not really connected to code as well, such as social contract.

Reply:
Otherwise the file looks good.